packages/react-ui/src/stories/common/Radio.stories.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { FormControl, FormControlLabel, FormLabel, Grid, Radio, RadioGroup } from '@material-ui/core';
+
+export default {
+  title: 'Common/Radio',
+  component: Radio,
+  argTypes: {
+    color: {
+      control: {
+        type: 'select',
+        options: ['default', 'primary', 'secondary']
+      }
+    },
+    disabled: {
+      control: {
+        type: 'boolean'
+      }
+    },
+    label: {
+      control: {
+        type: 'text'
+      }
+    }
+  }
+}
+
+const Template = ({ label, color = 'primary', checked = false, ...args }) => {
```

Comment:
For the Playground, don't set default values except for labels and similar props, so we can test the default state of the component